lib/finder_frontend.rb

```diff
@@ -93,7 +93,9 @@ def filter_params
     end
 
     def base_filter
-      finder.filter.to_h
+      finder.filter.to_h.reduce({}) { |memo, (k,v)|
+        memo.merge("filter_#{k}" => v)
```

Comment:
Rather than do this `reduce` dance twice, what about making `filter_params` do something like:

```ruby
def filter_params
  params
    .expect("keywords")
    .merge(base_filter)
    .reduce({}) …
end
```